en/flight_modes/flight_modes_summary.md

```diff
@@ -293,21 +316,26 @@ Multicopter hovers at the current GPS position and altitude.
  <td><a href="../flight_modes/follow_me.md">Follow Me</a></td>
  <td colspan="3">Auto</td>
  <td>Y</td>
- <td>
-Vehicle autonomously follows a user using an Android phone/tablet running QGC.
-</td>
+ <td>Vehicle autonomously follows a user using an Android phone/tablet running QGC.</td>
 </tr>
 
 <tr>
  <td><a href="../flight_modes/offboard.md">Offboard</a></td>
  <td colspan="3">Auto</td>
  <td>Y</td>
- <td>The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi).</td>
+ <td>Vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi).</td>
 </tr>
  
 </tbody></table>
 
 
+## VTOL
```

Comment:
@bkueng I added this section. I've pretty much said VTOL does the flight modes for MC/FW based on what config it is in. The exception being offboard, which is true for both configs, even though it is only in MC normally.